### DISCUSSION TITLE
mpv暂停挂在后台，操作系统冻结

### DISCUSSION THREAD

**Discussion by natural-harmonia-gropius** [hooke007/mpv_PlayKit#56] [mpv 通用]
最近发生过两次，都是有mpv在后台的情况，不排除是老黄驱动或者咖喱系统的锅。
就 https://github.com/mpv-player/mpv/issues/9424#issuecomment-985613681 这样操作，暂停了挂在后台，刷刷网页（包括b站视频页），忽然整个系统界面就冻结了。只有鼠标能动，鼠标移动很顺畅，疯狂的操作能出现指针右上角小圆圈。
ctrl+alt+del无效，ctrl+shift+win+b（不确定，可能雷蛇驱动死了或者太卡影响按键顺序导致无效？），只能硬重启解决。
cpu/gpu风扇没有狂转，基本排除这二者被占满导致卡死

**Comment by hooke007** [comment]
暂定为玄学问题
